src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/IndexViewModel.kt

```diff
@@ -20,8 +20,8 @@ data class IndexViewModel(@Serialise("reportsJson") val reports: List<ReportRowV
                           val showProjectDocs: Boolean,
                           val canSetPinnedReports: Boolean,
                           @Serialise("reportDisplayNamesJson") val reportDisplayNames: Map<String, String>?,
-                          val appViewModel: AppViewModel)
-    : AppViewModel by appViewModel
+                          val appViewModel: AppViewModel,
+                          val isReportRunner: Boolean? = false) : AppViewModel by appViewModel
```

Comment:
I think this is named `isRunner` elsewhere - would be good for it to be consistent. Also: does it make sense for it to be nullable?